src/openzaak/components/besluiten/models.py

```diff
@@ -43,42 +45,73 @@ class Besluit(AuditTrailMixin, APIMixin, models.Model):
         "organisatie die het besluit heeft vastgesteld.",
         db_index=True,
     )
-
-    _besluittype_url = models.URLField(
-        _("extern besluittype"),
+    _besluittype_base_url = models.ForeignKey(
+        Service,
+        null=True,
         blank=True,
-        max_length=1000,
-        help_text=_(
-            "URL-referentie naar extern BESLUITTYPE (in een andere Catalogi API)."
-        ),
+        on_delete=models.PROTECT,
+        related_name="besluit_besluittypen",
+        help_text="Basis deel van URL-referentie naar het extern BESLUITTYPE (in een andere Catalogi API).",
+    )
```

Comment:
I think I'd consider making this a custom field as well that bakes in the defaults of `null=True, blank=True, on_delete=models.PROTECT, to="zgw_consumers.Service"`